Folge_02/index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+
+<head>
+  <title>My first react application</title>
+</head>
+
+<body>
+  <h1>Hello World!</h1>
+  <div id="my-react-app"></div>
+
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  <script type="module" src="./lib/app.js"></script>
```

Comment:
Müsste das nicht `dist` sein? Irgendwas werfe ich hier gerade durcheinander, glaube ich 😉

Reply:
S.h. mein anderes Kommentar: in der Babel Doku ist das `lib` - daher verwende ich es in dieser Folge so.
Denke in der nächsten Folge mache ich das dann anders